tests/protos/options.proto

```diff
@@ -19,3 +19,7 @@ extend google.protobuf.FileOptions {
 extend google.protobuf.MethodOptions {
   optional string method_option = 20160518;
 }
+
+extend google.protobuf.ServiceOptions {
+  optional string service_level_option = 50001;
```

Comment:
(nit) looks like we've been using the int-date as the unique tag for options in this file (so use `20170125` instead of `50001`)